src/tasks.md

```diff
@@ -57,30 +57,31 @@ You can list all the `kwctl` options and subcommands by running the following co
 
 ```shell
 $ kwctl --help
-kwctl 0.2.4
-Flavio Castelli <fcastelli@suse.com>:Rafael Fernández López <rfernandezlopez@suse.com>
+kwctl 0.2.5
+Kubewarden Developers <kubewarden@suse.de>
 Tool to manage Kubewarden policies
 
 USAGE:
-    kwctl [FLAGS] <SUBCOMMAND>
+    kwctl [OPTIONS] <SUBCOMMAND>
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+OPTIONS:
+    -h, --help       Print help information
     -v               Increase verbosity
+    -V, --version    Print version information
 
 SUBCOMMANDS:
-    annotate       Adds Kubewarden metadata to a WebAssembly module
-    completions    Generates shell completions
-    help           Prints this message or the help of the given subcommand(s)
-    inspect        Inspects Kubewarden policy
-    manifest       Scaffolds a Kubernetes resource
+    annotate       Add Kubewarden metadata to a WebAssembly module
+    completions    Generate shell completions
+    digest         Fetch the digest of its OCI manifest
```

Comment:
Better to clarify what “its” refers to here. I assume we’re referring to the OCI artefact containing the policy? In which case, we can rephrase it as the below.

```suggestion
    digest         Fetches the digest of the artifacts  OCI manifest
```

Reply:
Agree, opened https://github.com/kubewarden/kwctl/pull/189.